### PR TITLE
WSO-436 - Resolve Errors - ucgy_core_filters

### PR DESCRIPTION
**Pull request by SandyVallee**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module [hunk 1]
```diff
@@ -183,58 +183,60 @@ function ucgy_core_filters_element_info_alter(&$type) {
...
-            // 'filtered HTML'.
-            $element['format']['format']['#default_value'] = 'filtered_html';
-            $element['format']['format']['#access'] = user_access('administer ckeditor');
-
-            // remove the 'help text' of the other formats
-            $element['format']['guidelines']['bold_and_italics'] = null;
-            $element['format']['guidelines']['full_html'] = null;
-            $element['format']['guidelines']['plain_text'] = null;
-            break;
+    if(isset($element['#field_name'])){
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module]: Need to add space between condition if and bracket "if (isset"
Also need a space between end of condition ")) {"

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module [hunk 2]
```diff
@@ -183,58 +183,60 @@ function ucgy_core_filters_element_info_alter(&$type) {
...
+      switch($element['#field_name']) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module]: Need space between switch and bracket "switch ("

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module [hunk 3]
```diff
@@ -183,58 +183,60 @@ function ucgy_core_filters_element_info_alter(&$type) {
...
+              // 'bold_and_italics'.
+              $element['format']['format']['#default_value'] = 'bold_and_italics';
+              $element['format']['format']['#access'] = user_access('administer ckeditor');
+
+              // remove the 'help text' of the other formats
+              $element['format']['guidelines']['filtered_html'] = null;
+              $element['format']['guidelines']['full_html'] = null;
+              $element['format']['guidelines']['plain_text'] = null;
+              continue;
+
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module]: space should be removed

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module (hunk 1 above)
Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module]: done

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module (hunk 2 above)
Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module]: done

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module (hunk 3 above)
Comment (SandyVallee) [reply sites/all/modules/ucalgary/ucgy_core/ucgy_core_filters/ucgy_core_filters.module]: done

**Pull request merged**